blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,258 @@
+
+[**🐳 Deploy and scale Sourcegraph with docker-compose**](#deploy-and-scale-sourcegraph-with-docker-compose)<br />
+
+[**📣 Improved campaign creation, management, and syncing performance**](#improved-campaign-creation-management-and-syncing-performance)<br />
+
+[**⚠️ Required migration for Kubernetes deployments**](#required-migration-for-kubernetes-deployments)<br />
+
+[**📈 Improved debuggability of Kubernetes deployments**](#improved-debuggability-of-kubernetes-deployments)<br />
+
+[**🚨 Monitoring and alerting added to the Sourcegraph instrumentation**](#monitoring-and-alerting-added-to-the-sourcegraph-instrumentation)<br />
+
+[**🧮 Estimate required resources for your Sourcegraph instance**](#estimate-required-resources-for-your-sourcegraph-instance)<br />
+
+[**🙅‍♀️ Repository management now can exclude GitHub forks/archives**](#repository-management-now-can-exclude-github-forksarchives)<br />
+
+[**🧠 LSIF improvements to help advertise?**](#lsif-improvements-to-help-advertise)<br />
+
+[**🛎 Aggregated search data added to Sourcegraph pings**](#aggregated-search-data-added-to-sourcegraph-pings)<br />
+
+[**🔎 Experimental smart search bar**](#experimental-smart-search-bar)<br />
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Permissions syncing improvements for GitLab and Bitbucket Server
+
+TODO: Video?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+The Sourcegraph permissions syncing has gotten more sophisticated to better handle repository permissions at scale. Rather than updating a user’s permissions when they log in and potentially blocking them from seeing any search results, Sourcegraph now syncs these permissions opportunistically, and as part of a background service. Users are now able to see results from their search queries immediately. Currently, this improvement is supported for GitLab and Bitbucket Server, with GitHub and other code hosts coming soon. 
+
+Enable this feature by adding `"permissions.backgroundSync": {"enabled": true}` in your site configuration settings. 
```

Comment:
@unknwon: Do we have documentation for admins about what happens when they turn this on? Since permissions take a while to sync, and we don't copy existing permissions, this would be a disruption of service, so I think we need some docs to guide admins through this process.

Reply:
@unknwon is going to add docs before this goes live tomorrow :)